packages/fxa-settings/src/pages/AccountRecoveryResetPassword/en.ftl

```diff
@@ -0,0 +1,8 @@
+password-link-expired = Reset password link expired
+password-link-damaged = Reset password link damaged
+create-new-password = Create new password
```

Comment:
```suggestion
# Header for form to create new password
create-new-password-header = Create new password
```

Reply:
This could easily be confused for text for something like a button or link to create a new password, so ID and comment needed for localizers.